### PR TITLE
[VSPC-162] If a module has a long name, it's display on the dashboard…

### PR DESCRIPTION
**Pull request by AvirupBiswas**
… gets adjusted now

# Guidelines for Pull Requests

If you haven't yet read our code review guidelines, please do so, You can find them [here](https://diging.atlassian.net/wiki/spaces/DIGING/pages/2256076801/Code+Review+Guidelines).

Please confirm the following by adding an x for each item (turn `[ ]` into `[x]`).

- [x] I have removed all code style changes that are not necessary (e.g. changing blanks across the whole file that don’t need to be changed, adding empty lines in parts other than your own code)
- [x] I am not making any changes to files that don’t have any effect (e.g. imports added that don’t need to be added)
- [x] I do not have any sysout statements in my code or commented out code that isn’t needed anymore
- [x] I am not reformatting any files in the wrong format or without cause. 
- [x] I am not changing file encoding or line endings to something else than UTF-8, LF
- [x] My pull request does not show an insane amount of files being changed although my ticket only requires a few files being changed
- [x] I have added Javadoc/documentation where appropriate
- [x] I have added test cases where appropriate
- [x] I have explained any part of my code/implementation decisions that is not be self-explanatory

## Please provide a brief description of your ticket 
If a module has a long name, it's display on the dashboard gets messed up.

After Fixing:
If a module has a long name, it's display on the dashboard gets adjusted now
  
## Anything else the reviewer needs to know?

... describe here ...

### CODE REVIEW

**Comment by jdamerow** [comment]
Make it so, Jenkins.

File: vspace/src/main/webapp/WEB-INF/views/staff/dashboard/dashboard.html [hunk 1]
```diff
@@ -46,8 +46,11 @@ <h4>Spaces</h4>
 			<h4>Modules</h4>
 			<div class="grid" style="margin-top: 10px;">
 				<div th:each="module,iterStat: ${recentModules}" th:if="${iterStat.index}<5" >
-                    <div class="btn-group">
-					<a th:href="@{'/staff/module/'+${module.id}}" class="btn btn-light" style="width: 10rem;">
+                    <div class="btn-group" >
+					<a th:href="@{'/staff/module/'+${module.id}}" class="btn btn-light" th:if="${module.name!=null && #strings.length(module.name)>20}" style="width:10rem;white-space: normal;overflow-wrap: break-word;">
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/staff/dashboard/dashboard.html:50]: why don't you just always apply the style (overflow-wrap, white-space)? why only for module names longer than 20?

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
Resolve conflicts please

**Comment by AvirupBiswas** [comment]
> Resolve conflicts please

The Module list shown in the dashboard has already been changed in VSPC-167 so this issue has already been handled there. So this ticket can be closed without any code change.

**Pull request closed without merging**